ui/src/dashboards/containers/DashboardPage.js

```diff
@@ -311,6 +284,28 @@ class DashboardPage extends Component {
     dashboardActions.putDashboardByID(dashboardID)
   }
 
+  handleSaveTemplateVariables = async templates => {
+    const {location, dashboardActions, dashboard} = this.props
+
+    try {
+      await dashboardActions.putDashboard({
+        ...dashboard,
+        templates,
+      })
+      const deletedTempVars = dashboard.templates.filter(
```

Comment:
will this not happen automatically when `dashboard` is updated in redux state, based on the `putDashboard`?